### ISSUE TITLE
npm test throw error at events.js:72

### ISSUE THREAD

**Issue by jlpoolen** [mapbox/mapbox.js#496]
I think I successfully built the project; though here is the pastebin of my build attempt and subsequent test:  http://pastebin.com/7DdQtrRM

here's the test with error:

> themis mapbox.js # npm test
> 
> > mapbox.js@1.3.1 test /usr/local/src/mapbox/mapbox.js
> > mocha-phantomjs test/index.html
> 
> events.js:72
>        throw er; // Unhandled 'error' event
>              ^
> Error: spawn ENOENT
>    at errnoException (child_process.js:980:11)
>    at Process.ChildProcess._handle.onexit (child_process.js:771:34)
> npm ERR! weird error 8
> npm ERR! not ok code 0
> themis mapbox.js #

**Comment by tmcw** [comment]
Do you have phantomjs installed?

**Comment by jlpoolen** [comment]
Not that I know of.  I'll attempt to install it now and update with my results.

**Comment by jlpoolen** [comment]
I did not have phantomJS installed and therefore installed it. 
(Note to Gentoo users, phantomJS is in portage, it's not a small install (took over 30 minutes on a quadcore 6 GB laptop!): emerge www-client/phantomjs)

 I recommend the instructions:

> Test with phantomjs:

might be modified to alert the user that a phantomJS is a prerequisite and/or needs separate installation.  Perhaps:

> Test with phantomJS (install separately):

I successfully ran the test.  Output at: http://pastebin.com/G8y7jGXV

Thank you!
